wm_2dnav/config/base_local_planner_params.yaml

```diff
@@ -33,10 +33,15 @@ TrajectoryPlannerROS:
   #The weighting for how much the controller should stay close to the path it was given. default 0.6
   pdist_scale: 0.3
   # The weighting for how much the controller should attempt to avoid obstacles. default 0.01
-  occdist_scale: 0.7
+  occdist_scale: 0.2
   # The weighting for how much the controller should attempt to reach its local goal, also controls speed. default 0.8
   gdist_scale: 0.5
   heading_scoring: false
   oscillation_reset_dist: 0.10
-   
-  
+
+  # Clean up the points
+  prune_plan: true
```

Comment:
Are those change related to the new repo structure or have been introduced by mistake?